components/identifier-generator/front/src/feature/components/LabelTranslations.tsx

```diff
@@ -20,10 +20,17 @@ const LabelTranslations: React.FC<LabelTranslationsProps> = ({labelCollection, o
 
   const onLabelChange = useCallback(
     (locale: string) => (label: string) => {
-      setValue({...value, [locale]: label});
-      onLabelsChange({...value, [locale]: label});
+      if (label.trim() !== '') {
+        setValue({...value, [locale]: label.trim()});
+        onLabelsChange({...value, [locale]: label});
+      } else {
+        const updatedLabels = {...labelCollection};
```

Comment:
Sorry i didn't see that we used value above, let's keep the consistency
```suggestion
        const updatedLabels = {...value};
```